src/config.py

```diff
@@ -1,11 +1,14 @@
 """training.py configurations"""
 
+import os
 import numpy as np
 from pathlib import Path
 
-MODEL = 'stage-2_bs16.pkl'
-MODEL_URL = 'https://drive.google.com/uc?id=1ClEUOwFhIOMNxvedps8WQN8Koy6NT5eI'
+PORT = int(os.environ.get("PORT", 5000))
```

Comment:
May wanna be more specific about which ports / hosts these are. Are they the frontend or backend?